### ISSUE TITLE
Typescript error (I assume)

### ISSUE THREAD

**Issue by svermaak** [jodit/jodit-react#98]
I am running TypeScript 4.0.3

![image](https://user-images.githubusercontent.com/12259728/93981231-52db0680-fd9d-11ea-831e-c10ec0f108c9.png)

**Comment by jeffzee** [comment]
@svermaak  same here. any solution or workaround? Thanks

**Comment by svermaak** [comment]
No. I have tried a new version of rush-stack-compiler-3.7 but same error. I believe the version of TypeScript is too low
https://www.npmjs.com/package/@microsoft/rush-stack-compiler-3.7
https://siddharthvaghasia.com/2019/06/09/how-to-upgrade-spfx-solution-to-latest-version/

**Comment by amirofy2002** [comment]
TypeScript error in C:/My Data/React/FitnessCenter/node_modules/jodit/index.d.ts(7,13):
'=' expected.  TS1005

     5 |  */
     6 | 
  >  7 | import type { IComponent, Nullable } from './src/types';
       |             ^
     8 | import { Jodit as Super } from './src/jodit';
     9 | 
    10 | export * from './src/types';

**Comment by gingfreeze** [comment]
ive fixed this issue by upgrading my typescript version to @4.0.3
hope this help

**Comment by harukaeru** [comment]
Same here. Actually, jodit seems to be created in **typescript 4**.
Unfortunately, we use typescript 3, and that triggers the errors. (I didn't prove it is really true. However, some errors are explicitly incompatible with features in typescript 3.)
I believe upgrading to typescript 4 will resolve these issues.

For me, since we cannot upgrade it for another reason, I declared a type definition for Jodit and used a CDN version. The following is an example.

```typescript
// typings/jodit.d.ts

// FIXME: Temporally defined. upgrading to typescript 4 and then delete the following.
interface IJodit {
  new (name: string, options: any)
}
declare const Jodit: IJodit
```
